src/main.rs

```diff
@@ -0,0 +1,62 @@
+extern crate clap;
+use clap::{App, Arg};
+use libwallet::{
+     self,
+     sr25519::{Pair, Public},
+     Pair as _, SimpleVault, Wallet,
+};
+use sp_core::crypto::{Ss58AddressFormat, Ss58Codec};
+
+#[async_std::main]
+async fn main() {
+     let matches = App::new("Wallet Generator")
+          .version("0.1.0")
+          .author("Daniel Olano <me@olanod.com>")
```

Comment:
We'll do this together ;)
```suggestion
          .author("Virto Team <we@virto.team>")
```